common/changes/office-ui-fabric-react/issue8063_2019-02-22-04-50.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Persona: Augment documentation for `optionalText` and `tertiaryText` to be more useful.",
+      "type": "patch"
```

Comment:
Aren't doc changes usually `none`?

Reply:
Doc changes of API surface is different from doc changes of internal code :)